oktaawscli/aws_auth.py

```diff
@@ -243,11 +285,11 @@ def __create_options_from(roles):
             # role[0] is the role arn, role[2] is the account alias
             options.append("[%s]: %s : %s" % (str(index + 1).ljust(2), role[2].ljust(27), role[0]))
         return options
-
     def __find_predefined_role_from(self, roles):
         # role_tuple[0] is the role arn
         found_roles = filter(lambda role_tuple: role_tuple[0] == self.role, roles)
         if not found_roles:
             return None
-        else:
-            return next(found_roles)
+        elif six.PY2:
+            return found_roles[0]
```

Comment:
It turns out that Python 2 doesn't work with using next(), but works with getting the first index of the array.